### PR TITLE
Add note for `serve --public` that every description is accessible

### PR DESCRIPTION
**Pull request by mschnitzer**
Supersedes #1670

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -4,6 +4,8 @@
   directory for the `build` command (gh#SUSE/machinery#1667)
 * Fix wrong hint for docker containers while running `analyze` command
   (gh#SUSE/machinery#1632)
+* Add a new hint when using the `--public` option for the `serve` command that it
+  makes every descriptions publicly available
```
Comment (aduffeck) [inline NEWS]: There's a small typo here (`every descriptions` -> `all descriptions`).

Comment (mschnitzer) [inline NEWS]: Whoops. Fixed ;-)

**Comment by aduffeck** [comment]
lgtm

**Comment by mschnitzer** [comment]
Superseded by #1681

**Pull request closed without merging**